<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/images/network_workgroup.png</normaloff>:/images/network_workgroup.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout_3" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="GroupTab" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="connectionTab" >
       <attribute name="title" >
        <string>Public Chat</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget" >
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" >
             <widget class="QTextBrowser" name="textBrowser" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLineEdit" name="lineEdit" />
            </item>
           </layout>
          </widget>
          <widget class="QListWidget" name="listWidget" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Peer" />
   <addaction name="actionSettings" />
   <addaction name="actionAbout" />
   <addaction name="actionQuit" />
  </widget>
  <action name="actionSettings" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/images/package_system_32.png</normaloff>:/images/package_system_32.png</iconset>
   </property>
   <property name="text" >
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/images/info_32.png</normaloff>:/images/info_32.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAdd_Peer" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/images/list_add_user_32.png</normaloff>:/images/list_add_user_32.png</iconset>
   </property>
   <property name="text" >
    <string>add Peer</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/images/exit32.png</normaloff>:/images/exit32.png</iconset>
   </property>
   <property name="text" >
    <string>quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
